src/queryRunner/utility/testExport.js

```diff
@@ -24,8 +21,8 @@ export function createTestFileContents(its){
 }
 
 function wrapIts(its){
-  const mochaWrap = 
-  `const request = require('request');
+  const mochaWrap = `
```

Comment:
Value is a lone tick?

Reply:
No, GitHub only shows the change -- I started the template on the next line to preserve it's formatting

Reply:
Good looks. Couldn't find the closing tick, so that worried me